doc/en/user/source/extensions/geofence-server/rest.rst

```diff
@@ -99,40 +102,52 @@ All filter parameters are optional.
      - Used for paging a list of rules. Specifies the number of entries per page. Leave out for no paging. If specified, ``page`` should also be specified.
    * - userName
      - string
-     - Filter rules on username (excludes all other specified usernames).
+     - Filter rules on username (excludes all other specific usernames).
    * - userAny
      - 0 or 1. 
-     - Specify whether rules for 'any' username are included or not.
+     - Specify whether rules matching any username are included or not.
    * - roleName
      - string
-     - Filter rules on rolename (excludes all other specified rolenames).
+     - Filter rules on rolename (excludes all other specific rolenames).
    * - roleAny
      - 0 or 1. 
-     - Specify whether rules for 'any' rolename are included or not.
+     - Specify whether rules matching any rolename are included or not.
+   * - ipaddress
+     - string
+     - Filter rules on IP address range (only select rules with an address range that includes the passed IP address).
+   * - ipaddressAny
+     - 0 or 1. 
+     - Specify whether rules matching any IP address are included or not.
+   * - date
+     - string
+     - Filter rules by date. Format is `yyyy-MM-dd`.
+   * - dateAny
+     - 0 or 1. 
+     - Specify whether rules with no data range defined are included or not
```

Comment:
I'm confused, the images above report a start and an end date... maybe the GUI composes a date range, but this documentation... is it for a the GUI or the REST API?
And if the REST API changes, I guess it also needs its own documentation update.

Reply:
The **Rule** defines the date range.
The **RuleFilter** sets a date as a filter.

The Rule will match if the filtering date is enclosed in the date range.

The code you are citing is in the paragraph "Filter Parameters".

Reply:
In the new PR, improved a lot the REST doc